### PR TITLE
remove stalled backups using cronjobs

### PR DESCRIPTION
**Pull request by rashamalek**
remove stalled backups using cronjobs

### CODE REVIEW

File: cron-ghe-backup [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # ghe prod backups
+0 */2 * * 1-6 application find /data/ghe-production-data/ -maxdepth 2 -mindepth 2 -name incomplete -atime +1  -printf "%h\0" | xargs -0 rm -Rf
```
Comment (lotharschulz) [inline cron-ghe-backup:2]: `find` has a flag `-delete` that could have been used as well. I assume you want to print to log file before deleting, correct?

Comment (rashamalek) [reply cron-ghe-backup:2]: No, because -delete flag cannot delete directories containing files

**Comment by lotharschulz** [comment]
👍

**Comment by rashamalek** [comment]
:+1:

**Comment by rashamalek** [comment]
@lotharschulz could you please also approve it in case it is ok?

**Review by lotharschulz — APPROVED** [review]

**Pull request merged**